### DISCUSSION TITLE
About the use of Cmake

### DISCUSSION THREAD

**Discussion by ghost** [libgit2/libgit2#5845] [General] [answered]
The README says libgit2 is c90. But it uses Cmake for building which we know is c++11. 

What is the point of this? Can't we have a POSIX Makefile and configure script?

**Comment by ethomson** [comment, marked as answer]
> The README says libgit2 is c90. But it uses Cmake for building which we know is c++11. 

Why is this a problem?

> What is the point of this? Can't we have a POSIX Makefile and configure script?

Well, we build on non-POSIX systems, would be my first thought.  But if by "configure script" you mean autoconf, then that's not something we're going to add.

**Reply by ghost** [reply]
Not autoconf, plain old shell script

**Reply by ethomson** [reply]
No.  We build and run on Windows.
